ChangeLog

```diff
@@ -1,3 +1,14 @@
+Changes to squid-4.0.24 (06 Mar 2018):
+
+	- Bug 4822: Build failure (-Wformat) where time_t is not long int
+	- Bug 4505: SMP caches sometimes do not purge entries
+	- TLS: GnuTLS implementation for listening ports and client connections
+	- TPROXY: Fix clientside_mark and client port logging
+	- Native FTP: Fix "Cannot assign requested address" with TPROXY
+	- SSL-Bump: Fix authentication with types other than Basic
```

Comment:
FYI: The far more prevalent, historically accurate, and slightly more logical spelling is SslBump. This commit does not introduce this (minor) problem to ChangeLog, but it can fix it.

Reply:
Historically it has been a mix. "SSL-Bump" is to refer to the 'project' grouping I have for these patches generic patches. References to "SslBump" are mostly referring to the original 3.0/3.1 code functionality which went in by that name (or the way people titled their commits). I have been referring to the overall feature set as "SSL-Bump" since the mimic and generation features were added onto the "SslBump" feature.

FYI: The reason behind using that different name has been to stop new people wanting help going to the "SslBump" wiki page as their first stop for Squid-3.4+ configuration info. Big red deprecation warnings on the page did not help, but this subtle change to the terminology name being searched for in Google et al seems to have worked.

Reply:
> "SSL-Bump" is to refer ...

`SSL-Bump` should not be used (for the reasons I have already provided). One cannot justify using a bad or inferior term by describing how it is to be used, by ignoring conflicting usages, or by mentioning bad wiki organization ideas, etc., etc. No need to respond though -- I have no time to fight this nonsense.